### ISSUE TITLE
comments

### ISSUE THREAD

**Issue by 20150075** [EngDinaSayed/Software-Engineering---Phase-3#4]
there exist some functions and classes need some comments to be able to understand what it should do

**Comment by EngDinaSayed** [comment]
I agree with this opinion as everyone has his own coding style and to stop misunderstanding which may cause problems, waste in time and the other staff that we don't want to pump in.
